templates/telco/cnf.html

```diff
@@ -0,0 +1,469 @@
+{% extends "templates/one-column.html" %}
+
+{% block title %}CNF | Kubernetes{% endblock %}
+
+{% block meta_description %}Cloud Native Network Functions infrastructure with Charmed Kubernetes{% endblock %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1IpOPiWidozQ7w1hyM52yQn1gSwHODsItn1ZnMQ0retw/edit#{% endblock meta_copydoc %}
+
+{% block content %}
+
+<section class="p-strip is-bordered u-no-padding--top u-no-padding--bottom">
+  <div class="p-strip--suru-topped">
+    <div class="row u-equal-height u-vertically-center">
+      <div class="col-7">
+        <h1>
+          Kubernetes for CNF
+        </h1>
+        <p class="p-heading--3">
+          Cloud native network functions
+        </p>
+        <p>
+          CNF is a network function designed and implemented to run inside containers. CNFs follow cloud native architectural and operational principles including K8s lifecycle management, model driven operations, agility, resilience, and observability.
+        </p>
+        <p>
+          <a href="/telco/contact-us" class="p-button--positive js-invoke-modal">Contact us about Kubernetes</a>
```

Comment:
This contact us is not going anywhere and it's not opening the interactive form either.
When you click on it, it opens up a 404 page at the bottom

Reply:
Forgot to add in the description that I am waiting for the main telco page to be merged so we can test it. I will add a blocked label to this for now.

Reply:
ok, but the weird thing is that `href` is displaying a 404 page at the bottom...